tests/kafkatest/services/monitor/jmx.py

```diff
@@ -124,7 +124,7 @@ def read_jmx_output(self, idx, node):
 
         for name in object_attribute_names:
             aggregates_per_time = []
-            for time_sec in xrange(start_time_sec, end_time_sec + 1):
+            for time_sec in xrange(start_time_sec, end_time_sec + int(self.report_interval/100) + 1):
```

Comment:
Not sure why this change? Can you elaborate?

Reply:
The old code assumes that the report interval is always 1 second, and hence hard code it here, whereas I tried to make it configurable and in practice making the reporting interval to be much larger, and hence need to make this change. But we still keep the `+ 1` in case the report interval is configured to be smaller than 1 sec. Does that make sense?

BTW it should be `1000` in the denominator than `100` ..

Reply:
Not sure if we need to change the end of the range, but rather the "step size" of the `xrange` to change from `1` to `report_interval` ?